staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```
@@ -213,8 +213,11 @@ func (o *ScaleOptions) RunScale() error {
 		if err == nil {
```

It's preferred to fail fast, so 
```go
if err != nil {
    return err
}
infos = append(infos, info)
```